shardingsphere-scaling/shardingsphere-scaling-core/pom.xml

```diff
@@ -104,5 +104,10 @@
             <version>${curator.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
Why is it needed here?

Reply:
This is added here as in the test we use, `ReflectionTestUtils` for mocking private `Properties` defined in SingleIdleAutoSwitchAlgorithm.

Reply:
Thanks @kaplem , let's wait for all the CI tests.